components/landing/process-step.vue

```diff
@@ -0,0 +1,38 @@
+<template functional>
+  <div class="container">
+    <img :src="props.img" />
+    <div class="description">
+      <h3 class="title">{{ props.title }}</h3>
+      <p class="subtitle">{{ props.subtitle }}</p>
+      <nuxt-link
+        v-if="props.buttonLink"
+        :to="props.buttonLink"
+        class="btn filled green shadow-1"
+      >
+        {{ props.buttonText }}
+        <img v-if="props.external" src="images/landing/external-link.svg" class="ml-1" />
+      </nuxt-link>
+      <strong v-else>{{ props.buttonText }}</strong>
+    </div>
+  </div>
+</template>
+
+<script>
+  export default {
+    props: {
+      img: String,
+      title: String,
+      subtitle: String,
+      buttonLink: {
+        type: String,
+        default: '#',
```

Comment:
Make default be `'/'` cause we're not in plain html and can do this

Reply:
I don't understand. Why isn't `'#'` valid? It does the job of not making the link go anywhere

Reply:
You are right, but why do weed this hash in the url?

Reply:
because we don't want it to redirect anywhere. making it `/` will cause is to redirect to the root. For example, assume our site got hosted at "https://my.university.innopolis.ru/innopoints"; you would want the button (if not assigned a URL, which shouldn't be the case anyways!) to keep it at the same page, not go back to the dashboard